integration-cli/docker_cli_daemon_test.go

```diff
@@ -1804,3 +1804,44 @@ func (s *DockerDaemonSuite) TestDaemonStartWithDefalutTlsHost(c *check.C) {
 		c.Fatalf("docker version should return information of server side")
 	}
 }
+
+// Test Daemon starts when the repositories-aufs file is removed
+func (s *DockerDaemonSuite) TestDaemonStartWithNoRepoConf(c *check.C) {
```

Comment:
Tests run not only on aufs. You should skip them if `s.d.storageDriver != "aufs"`.